src/main/resources/com/google/api/codegen/nodejs/version_index.snip

```diff
@@ -42,25 +42,20 @@
     * Create an {@service.clientName} with additional helpers for common
     * tasks.
     *
-    * @@param {Object=} opts - The optional parameters.
-    * @@param {String=} opts.servicePath
-    *   The domain name of the API remote host.
-    * @@param {number=} opts.port
-    *   The port on which to connect to the remote host.
-    * @@param {grpc.ClientCredentials=} opts.sslCreds
-    *   A ClientCredentials for use with an SSL-enabled channel.
-    * @@param {Object=} opts.clientConfig
-    *   The customized config to build the call settings. See
-    *   {@@link gax.constructSettings} for the format.
+    {@comments(@service.doc.lines)}
+    *
+    * @@param {Object=} options - The optional parameters.
```

Comment:
Would it be possible to change this line to match this: https://github.com/GoogleCloudPlatform/google-cloud-node/blob/d152416b48928ab13a17f710425266ed1b82dce0/packages/video-intelligence/src/index.js#L52 -- that link is crucial for information about authentication and other basics.